flocker/cli/test/test_cli_script.py

```diff
@@ -12,7 +13,7 @@ class FlockerCLITests(FlockerScriptTestsMixin, TestCase):
     command_name = u'flocker'
 
 
-class CLIOptionsTests(StandardOptionsTestsMixin, SynchronousTestCase):
```

Comment:
Good catch!

Probably the "right" way to do it is the same way we do our interface tests for other things (e.g. `make_iblockdeviceapi_tests`). I don't quite have the energy for that right now, but I've filed https://clusterhq.atlassian.net/browse/FLOC-3794 and put it on the board. If no one picks it up before tomorrow's British morning, I'll have a go. In the mean time, I'll add a comment.